widgetry/src/backend_glow_wasm.rs

```diff
@@ -69,14 +64,18 @@ pub fn setup(
     // First try WebGL 2.0 context.
     // WebGL 2.0 isn't supported by default on macOS Safari, or any iOS browser (which are all just
     // Safari wrappers).
```

Comment:
BTW webGL2 is supported since Safari 15 (https://caniuse.com/webgl2) released a year ago (about 2% of users, according to that same site).

Reply:
Ah, awesome! I'll make a followup to cleanup the webgl1 stuff -- it'd be nice to maintain less copies of the shaders for sure.